### PR TITLE
DAS-2051: Adds environment variable option for AWS_ACCESS_TOKEN

### PR DESCRIPTION
**Pull request by flamingbear**
## Description

This allows a user with only short term credentials to run the regression tests.

## Jira Issue ID

None IP sprint found bug and squashed.

## Local Test Steps

Pull this branch and rebuild one or more regression test images.

Set up your environment.

``` shell
❯ export HARMONY_HOST_URL=https://harmony.sit.earthdata.nasa.gov
❯ export EDL_USER=<your username>
❯ export EDL_PASSWORD=<your password>
```

Also go to Kion and get a set of short-term access keys.

``` shell
❯ export AWS_ACCESS_KEY_ID=<key>
❯ export AWS_SECRET_ACCESS_KEY=<secret>
❯ export AWS_SESSION_TOKEN=<also secret>
``` 

Run one or more of the images to verify that you can indeed run the regression tests against short-term keys.

## PR Acceptance Checklist
* [N/A] Acceptance criteria met
* [N/A] Tests added/updated (if needed) and passing
* [N/A] Documentation updated (if needed)

### CODE REVIEW

File: test/swath-projector/version.txt
Comment (lyonthefrog) [inline test/swath-projector/version.txt:1]: 0.1.4 -> 0.1.5?

**Comment by lyonthefrog** [comment]
Why do we have to increment the version number for every test?

**Comment by lyonthefrog** [comment]
I believe that the only test that requires AWS access keys is `n2z`. Does this affect other tests when it's set?

File: test/Dockerfile [hunk 1]
```diff
@@ -10,6 +10,7 @@ ENV env_notebook=$notebook
 
 ENV AWS_SECRET_ACCESS_KEY=no-access
 ENV AWS_ACCESS_KEY_ID=no-access
+ENV AWS_ACCESS_TOKEN=no-access
```
Comment (lyonthefrog) [inline test/Dockerfile:13]: I think this should be AWS_SESSION_TOKEN, right?

**Review by lyonthefrog — CHANGES_REQUESTED** [review]

**Comment by lyonthefrog** [comment]
Unfortunately I'm still seeing a failure for the `n2z` run: `./run_notebook n2z`.

image

**Comment by flamingbear** [comment]
> Why do we have to increment the version number for every test?

We updated the dockerfile itself to add the ability to take a AWS_SECRET_TOKEN.  

That said, you're correct that only n2z uses that token and is the only version that needs to be bumped.  Also, you're correct that this didn't fix n2z, the one I had running took forever so I thought it was working.  Give me 10 min. Sorry

**Comment by flamingbear** [comment]
Ok. Sorry again, @lyonthefrog.  This should be a better version. and ready for re-review.

**Comment by lyonthefrog** [comment]
The tests pass now! `n2z` specifically, these are the results for using both a short-term access key and a long-term access key:

```
» ./run_notebooks.sh n2z

Running regression tests
Test suite n2z starting
running test with ghcr.io/nasa/regression-tests-n2z:latest
Waiting for n2z.
Input Notebook:  n2z/N2Z_Regression.ipynb
Output Notebook: /workdir/output/n2z/Results.ipynb
Working directory: n2z
Executing:   0%|          | 0/16 [00:00<?, ?cell/s]Executing notebook with kernel: python3
Executing: 100%|██████████| 16/16 [04:44<00:00, 17.78s/cell]
Test suite n2z succeeded
Tests completed (passed)
```

**Review by lyonthefrog — APPROVED** [review]

**Pull request merged**